rialto_airflow/harvest/dimensions.py

```diff
@@ -16,10 +16,10 @@
 def dois_from_orcid(orcid):
     logging.info(f"looking up dois for orcid {orcid}")
     q = """
-        search publications where researchers.orcid_id = "{}"
+        search publications where researchers.orcid_id = "{}" and year in [2018:{}]
```

Comment:
Why are we limiting to publications between 2018-2024. This seems like something we will forget about, and we will quietly stop getting new data?

Reply:
The thought was that since Rochelle only cared about pubs after 2017, we could speed up the run time by filtering in the query but I don't think it actually did speed things up so I pulled it out.